angio-frontend/src/modules/user/components/AccountActivationForm.vue

```diff
@@ -0,0 +1,174 @@
+                                v-bind:label="$t('user.component.activation.activationFormStep.newPassword.field')"
+                                v-bind:rules="[v => !!v || $t('user.component.activation.activationFormStep.newPassword.validation.NotEmpty'),
+                                    v => v.length >= 8 || $t('user.component.activation.activationFormStep.newPassword.validation.MinLength'),
+                                    form.newPassword === newPasswordRepeat || $t('user.component.activation.activationFormStep.newPassword.validation.DoesNotMatch')]"
+                                type="password"
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-text-field
+                                v-model="newPasswordRepeat"
+                                v-bind:label="$t('user.component.activation.activationFormStep.newPasswordRepeat.field')"
+                                v-bind:rules="[v => !!v || $t('user.component.activation.activationFormStep.newPassword.validation.NotEmpty'),
+                                    v => v.length >= 8 || $t('user.component.activation.activationFormStep.newPassword.validation.MinLength'),
+                                    form.newPassword === newPasswordRepeat || $t('user.component.activation.activationFormStep.newPassword.validation.DoesNotMatch')]"
+                                type="password"
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-expand-transition>
+                            <ul v-if="displayError" data-test-id="formMsg__list">
+                                <li class="red--text body-1">
+                                    {{ $t('user.component.activation.activationFormStep.error') }}
+                                </li>
+                            </ul>
+                        </v-expand-transition>
+                    </v-form>
+                </v-card-text>
+            </v-window-item>
+
+            <v-window-item v-bind:value="2">
+                <v-card-title class="headline font-weight-regular justify-space-between">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.title') }}</span>
+                    <v-spacer></v-spacer>
+                </v-card-title>
+                <v-card-text class="subheading font-weight-light">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.content') }}</span>
```

Comment:
Replace with icon (name: `done`, size: `x-large`) and short text description (less than 50 characters). Icon [doc](https://vuetifyjs.com/ru/components/icons).

Reply:
Done